File: Typescript/Practice/26072022/medium/medium_perimeters.py
#!/usr/bin/env python

def perimeter(l, num):
  result = (l == 's') * (4 * num) + (l == 'c') * (6.28 * num)
  return result

print(perimeter("s", 7))
print(perimeter("c", 4))
print(perimeter("c", 9))

File: Typescript/Practice/medium/medium_perimeters.ts
#!/usr/bin/env npx ts-node

function perimeter (l: string, num: number): number {
  let result = (l === 's') ? (4 * num) : (6.28 * num);
  return result
};

console.log(perimeter("s", 7));
console.log(perimeter("c", 4));
console.log(perimeter("c", 9));
